server/app/services/queryHandler/QueryHandler.java

```diff
@@ -91,13 +95,25 @@ private JsonNode processResponse(JsonNode responseBody, String questionMapping,
     return jsonNode;
   }
 
+  private JsonNode processResponse(String questionMapping){
+
+    JsonNode jsonNode = null;
+    try {
+      jsonNode = services.languageProcessor.TaskMap.questionMapping(questionMapping);
+    } catch (Exception e){
+      e.getMessage();
+    }
+
+    return jsonNode;
+  }
+
   public JsonNode parseErrorToJson(String message) {
 
-    Response response = new Response();
-    response.status = "fail";
-    response.message = message;
+    BotResponse botResponse = new BotResponse();
+    botResponse.status = "fail";
+    botResponse.message = message;
 
-    JsonNode msg = Json.toJson(response);
+    JsonNode msg = Json.toJson(botResponse);
```

Comment:
This can use `botResponseToJson` method without repeating the BotResponse object construction.